js/src/product-feed/issues-table-card/index.js

```diff
@@ -112,71 +78,8 @@ const IssuesTableCard = () => {
 						{ actions }
 					</div>
 				</CardHeader>
-				<CardBody size={ null }>
-					{ ! hasFinishedResolution && (
-						<TablePlaceholder headers={ headers } />
-					) }
-					{ hasFinishedResolution && ! data && (
-						<EmptyTable headers={ headers } numberOfRows={ 1 }>
-							{ __(
-								'An error occurred while retrieving issues. Please try again later.',
-								'google-listings-and-ads'
-							) }
-						</EmptyTable>
-					) }
-					{ hasFinishedResolution && data && (
-						<Table
-							headers={ headers }
-							rows={ data.issues.map( ( el ) => {
-								return [
-									{
-										display:
-											el.severity === 'warning' ? (
-												<WarningIcon />
-											) : (
-												<ErrorIcon />
-											),
-									},
-									{ display: el.product },
-									{ display: el.issue },
-									{
-										display: (
-											<AppDocumentationLink
-												context="issues-to-resolve"
-												linkId={ el.code }
-												href={ el.action_url }
-											>
-												{ el.action }
-											</AppDocumentationLink>
-										),
-									},
-									{
-										display: el.type === 'product' && (
-											<EditProductLink
-												productId={ el.product_id }
-												eventName="gla_edit_product_issue_click"
-												eventProps={ {
-													code: el.code,
-													issue: el.issue,
-												} }
-											/>
-										),
-									},
-								];
-							} ) }
-						/>
```

Comment:
This is now in its own component in `js/src/product-feed/issues-table-card/issues-table-data.js`